fastestimator/util/util.py

```diff
@@ -76,3 +78,10 @@ def get_num_devices() -> int:
     except:
         num_devices = 1
     return num_devices
+
+
+def lcms(*numbers):
+    def lcm(a, b):
```

Comment:
why hide this func def inside the parent func def?

Reply:
`lcm` is only intended to be used by `lcms` for recursion purpose.   The user only need to use the `lcms`

Reply:
that doesn't seem like a particularly good reason to do this. Do you know if there are any performance ramifications of defining functions inside other functions?

Reply:
it can keep the lcm private such that user won't use the wrong function.